editor/src/clj/editor/gl/shader.clj

```diff
@@ -621,10 +642,9 @@ of GLSL strings and returns an object that satisfies GlBind and GlEnable."
                       (range (gl-shader-parameter gl program GL2/GL_ACTIVE_ATTRIBUTES)))
 
                 uniform-infos
-                (into {}
+                (into []
```

Comment:
Uniform infos are just an array now since we need to do the lookup based on the :name and the :canonical-name to find a match.

Reply:
This seems like it would be slower.

Can't we simply put the same `uniform-info` under both keys in the `uniform-infos` map? I.e.
```clj
uniform-infos
(into {}
      (mapcat (fn [^long uniform-index]
                (let [uniform-info (uniform-info gl program uniform-index)]
                  [(pair (:name uniform-info) uniform-info)
                   (pair (:canonical-name uniform-info) uniform-info)])))
      (range (gl-shader-parameter gl program GL2/GL_ACTIVE_UNIFORMS)))
```